src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -27,7 +27,9 @@ namespace OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation
     /// </summary>
     internal static class AWSLambdaUtils
     {
-        internal const string ActivitySourceName = "Amazon.AWS.AWSLambdaInstrumentation";
+        internal static string ActivitySourceName =
+            typeof(AWSLambdaUtils).Assembly.GetName().Name.Replace(".Contrib", string.Empty);
```

Comment:
What is the expected result here? OpenTelemetry.Instrumentation.AWSLambda? Why do we remove the .Contrib @Kielek (https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/534#discussion_r9298623909)?

Reply:
@Oberon00, there is a decision to remove `Contrib` infix for all packages. Only AWS related do not follow this change, but @srprash mentioned that he would like to do this when major version will be increased. See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/196#issuecomment-1092161046

The second thing to consider is task to use common pattern name for all activity source https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/484

If we decide to change the activity source for this package, it will be good to make the change only once. So my proposal is to use current package name without `.Contrib`.

Reply:
Since we had a breaking change in #408, I think it would make sense to merge this. Can you please resolve merge conflicts @Kielek?

Reply:
@samimusallam is the branch owner. So he should do this. Probably also @open-telemetry/dotnet-contrib-maintainers have possibility to do this.\
If it will be not possible, we should recreate PR from the fork where we can modify branches.

Reply:
Sorry, for some reason I thought you submitted the PR and did not check. So of course I wanted to ask the PR author @samimusallam to resolve the conflicts.

Reply:
@Oberon00 conflicts resolved :)